src/main/java/com/java_essential/examples/sagandikovs/classwork/Main.java

```diff
@@ -0,0 +1,4 @@
+package com.java_essential.examples.sagandikovs.classwork;
```

Comment:
почему Вы в екзамплы полезли?
екзамплы ето мой пекедж
```suggestion
package com.java_essential.sagandikovs.classwork;
```